Gemfile.lock

```diff
@@ -45,12 +45,14 @@ GEM
       json
       simplecov
       url
+    coderay (1.1.1)
     concurrent-ruby (1.0.2)
     debug_inspector (0.0.2)
     diff-lcs (1.2.5)
     docile (1.1.5)
     erubis (2.7.0)
     ffi (1.9.14)
+    geometry (6.5)
```

Comment:
I just looked into https://github.com/DanielVartanov/ruby-geometry and I think it is much more solid and probably way faster than the `geometry` library we use. And at the moment we do not use that much from the library after all. 
We should switch to ruby-geometry

The harderst part is probably generating a seed polygon :) but we can probably just enter three point by random into a normal polygon.